drivers/generic/generic.go

```diff
@@ -127,13 +122,14 @@ func (d *Driver) Create() error {
 			" future will require the ssh agent to contain the appropriate key.")
 	} else {
 		log.Info("Importing SSH key...")
-		// TODO: validate the key is a valid key
-		if err := mcnutils.CopyFile(d.SSHKey, d.GetSSHKeyPath()); err != nil {
-			return fmt.Errorf("unable to copy ssh key: %s", err)
+
+		d.SSHKeyPath = d.ResolveStorePath("id_rsa")
```

Comment:
As a side note:  I think it's not strictly the case that an SSH key will be named `id_rsa` (the key might use another algo than RSA), but RSA is so massively popular vs. the others that we might be OK to just assume it.